.github/workflows/hail_update.yaml

```diff
@@ -8,6 +8,9 @@ on:
       hail_version:
         description: 'Hail version (as uploaded to PyPI)'
         required: true
+      hail_sha:
```

Comment:
Adding more inputs makes this more difficult to manually release, eg: if we manually release after updating SM. It also makes it impractical to automatically trigger a release from SM repo, because it would need to find the input for two hail variables.

The hail team have a deploy script which automatically applies a change and triggers a pull request, which might be one way to do this:

https://github.com/hail-is/hail/blob/660616e51f76ca244c2d4a6081ed169d740c10cc/hail/scripts/deploy.sh#L141-L148

Reply:
Wouldn't that be equivalent to a dummy commit to our Hail repo, as that would also trigger https://github.com/populationgenomics/hail/blob/main/.github/workflows/package.yaml, which in turn runs the analysis-runner workflow with the right inputs?

Reply:
As discussed over Zoom, we could try to replace this with a mechanism that gets the HEAD SHA + version by checking out the Hail repository. However, now that @vladsaveliev suggested that we might actually want to use the standard `hail` package in PyPI instead of `cpg-hail`, I'm not so sure anymore...